models/user.go

```diff
@@ -101,7 +101,6 @@ type User struct {
 	LoginName   string
 	Type        UserType
 	OwnedOrgs   []*User       `xorm:"-"`
-	Orgs        []*User       `xorm:"-"`
```

Comment:
We have to remove `.Orgs` or else we have to upgrade the type to have `NumRepos` as well, which would propagate the complexities unnecessarily.